src/pages/settings/Subscription/CardSection/CardSection.tsx

```diff
@@ -34,7 +34,18 @@ function CardSection() {
             isCentralPane
             titleStyles={styles.textStrong}
             subtitleMuted
-            banner={BillingBanner}
+            banner={
+                shouldShowBanner && (
+                    <SubscriptionBillingBanner
+                        title={title}
+                        subtitle={subtitle}
+                        isTrialActive={false}
```

Comment:
question, Is it always false?

Reply:
It turned out this prop is not necessary, I removed it